packages/ramda-extension/src/between.js

```diff
@@ -0,0 +1,29 @@
+import { curry } from 'ramda';
+/**
+ * See if an number (`val`) is within an array of two numbers ('list').
+ *
+ * @func
+ * @category Type
+ * @param {Number} a Starting value
+ * @param {Number} b Ending value
+ * @param {Number} val The value to test
+ * @return {Boolean}
+ * @example
+ *
+ *      R_.between(4, 1, 5); //=> true
+ *      R_.between(2.1, 3, 8); //=> false
+ *      R_.between(100.1, 100.1, 102); //=> true
+ *      R_.between(100.1)(100.1, 102); //=> true
+
+
+ */
+
+
+const between = curry((val, a, b) => {
```

Comment:
functions should be written as that the data to be operated on comes last. So (a, b, val)

Reply:
Maybe there should be better naming - like min, max, value.

Reply:
@lukasduspiva not needed - see pointfree implementation bellow